Mathlib/Topology/ContinuousFunction/Bounded.lean

```diff
@@ -1401,32 +1401,31 @@ functions from `α` to `β` is naturally a module over the algebra of bounded co
 functions from `α` to `𝕜`. -/
 
 
-instance hasSmul' : SMul (α →ᵇ 𝕜) (α →ᵇ β) :=
-  ⟨fun (f : α →ᵇ 𝕜) (g : α →ᵇ β) =>
+instance hasSmul' : SMul (α →ᵇ 𝕜) (α →ᵇ β) where
+  smul f g :=
     ofNormedAddCommGroup (fun x => f x • g x) (f.continuous.smul g.continuous) (‖f‖ * ‖g‖) fun x =>
       calc
         ‖f x • g x‖ ≤ ‖f x‖ * ‖g x‖ := norm_smul_le _ _
         _ ≤ ‖f‖ * ‖g‖ :=
           mul_le_mul (f.norm_coe_le_norm _) (g.norm_coe_le_norm _) (norm_nonneg _) (norm_nonneg _)
-        ⟩
 #align bounded_continuous_function.has_smul' BoundedContinuousFunction.hasSmul'
 
 instance module' : Module (α →ᵇ 𝕜) (α →ᵇ β) :=
   Module.ofCore <|
-    { smul := (· • ·)
-      smul_add := fun _ _ _ => ext fun _ => smul_add _ _ _
+    { smul_add := fun _ _ _ => ext fun _ => smul_add _ _ _
       add_smul := fun _ _ _ => ext fun _ => add_smul _ _ _
       mul_smul := fun _ _ _ => ext fun _ => mul_smul _ _ _
       one_smul := fun f => ext fun x => one_smul 𝕜 (f x) }
 #align bounded_continuous_function.module' BoundedContinuousFunction.module'
 
-theorem norm_smul_le (f : α →ᵇ 𝕜) (g : α →ᵇ β) : ‖f • g‖ ≤ ‖f‖ * ‖g‖ :=
-  norm_ofNormedAddCommGroup_le _ (mul_nonneg (norm_nonneg _) (norm_nonneg _)) _
-#align bounded_continuous_function.norm_smul_le BoundedContinuousFunction.norm_smul_le
-
-/- TODO: When `NormedModule` has been added to `Analysis.NormedSpace.Basic`, the above facts
-show that the space of bounded continuous functions from `α` to `β` is naturally a normed
+/- TODO: When `NormedModule` has been added to `Analysis.NormedSpace.Basic`, this
+shows that the space of bounded continuous functions from `α` to `β` is naturally a normed
 module over the algebra of bounded continuous functions from `α` to `𝕜`. -/
```

Comment:
Are we ever doing this? I thought we were purposefully avoiding it.

bors d+

Reply:
Yeah, I suspect we're not, but I wasn't really sure what to do with this comment and wanted to just declare fixing it out of scope.

Reply:
Do you want me to delete it?